qmp/libvirtgo_linux_test.go

```diff
@@ -0,0 +1,357 @@
+// Copyright 2016 The go-qemu Authors.
```

Comment:
All tests need to make use of interfaces instead of swapping out functions at the package level.